src/Configuration/UnleashContext.php

```diff
@@ -15,6 +15,7 @@ public function __construct(
         private ?string $currentUserId = null,
         private ?string $ipAddress = null,
         private ?string $sessionId = null,
+        private ?string $environment = null,
```

Comment:
Another BC break, new parameters can only be added at the end of the method

Reply:
I'll do it